lib/access_import/accessors/personen_rolle.rb

```diff
@@ -8,39 +10,41 @@ def sanitize_record(rec)
     rec = parse_int_fields(rec, :b_EinführungsKurs, :b_Interesse, :b_SpesenVerzicht,
       :fk_Hauptperson, :fk_Kostenträger, :pk_PersonenRolle, :z_AnzErw, :z_AnzKind,
       :z_Familienverband, :z_Rolle)
-    rec[:rolle] = map_rolle(rec[:z_Rolle])
+    rec[:rolle] = ACCESS_TO_OWN_ROLES_MAP[rec[:z_Rolle]]
     rec
   end
 
-  def map_rolle(z_rolle)
-    return 'Volunteer' if z_rolle == 1
-    return 'Client' if z_rolle == 2
-    return 'Animator' if z_rolle == 3
-    return 'Participant' if z_rolle == 4
-    nil
-  end
+  ACCESS_TO_OWN_ROLES_MAP = [
+    nil,
+    'Volunteer',
+    'Client',
+    'Animator',
+    'Participant'
+  ].freeze
```

Comment:
It probably would be safer to use a hash here, as the index number is a static key. When e.g. someone adds a new entry somewhere in this array a bug will be introduced as the index keys change.